examples/react/basic/tsconfig.json

```diff
@@ -1,20 +1,21 @@
 {
   "compilerOptions": {
-    "target": "ESNext",
-    "useDefineForClassFields": true,
-    "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": false,
-    "skipLibCheck": true,
-    "esModuleInterop": false,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "esModuleInterop": false,
+    "exactOptionalPropertyTypes": true,
```

Comment:
Are we making some useful changes to the example?

Reply:
I was literally wondering if this is a change or just a re-ordering. Looks like we're enabling exactOptionalPropertyTypes.

Reply:
ahh ok: it's not functionally (sorting aside) a change because the codebase already follows exactOptionalProperties, I just hadn't realized that this wasn't on for this example (I tried to keep the examples simpler and more portable by not inheriting from the base).